resources/js/components/search.js

```diff
@@ -5,18 +5,30 @@ var architects = new Bloodhound({
 	queryTokenizer: Bloodhound.tokenizers.whitespace,
 	datumTokenizer: Bloodhound.tokenizers.whitespace,
 	remote: {
-		url: '/api/search-sugestions/?search=%QUERY',
-		wildcard: '%QUERY',
-		transform: ({architects}) => architects
+		url: '/api/search-sugestions/',
+		prepare: function (query, settings) {
+            var locale = $('html').attr('lang');
+            settings.url += '?search='+query;
+            settings.url += '&locale='+locale;
+            console.log(settings.url);
+			return settings;
+        },
+        transform: ({architects}) => architects,
```

Comment:
How about getting the `locale` higher up? Then you could do
```suggestion
		url: `/api/search-sugestions/?locale=${locale}&search=%QUERY`,
		wildcard: '%QUERY',
		transform: ({architects}) => architects,
```

BTW, You don't have to go with my suggestion, but I think you'll still have to `encodeURIComponent()` your `query` if you don't use the `wildcard` option